lib/validate.js

```diff
@@ -8,7 +8,7 @@ const _ = require('lodash');
 
 function getEntryForFunction(serverlessFunction) {
 	const handler = serverlessFunction.handler;
-	const handlerFile = /(.*)\..*?$/.exec(handler)[1] + '.js';
+	const handlerFile = /(.*)\..*?$/.exec(handler)[1];
```

Comment:
At least somewhere under the servicePath. The handlerBaseName contains the path elements relative to the handler. So, for example, it could be `handlers/shop/order/handler`, but we can assume that handler files are never outside the service folder (of course the user can use symlinks to access files outside the service directory, but checking if the baseName + extension exists, will automatically reveil the correct results in this case)